pkg/config/libpodConfig.go

```diff
@@ -224,6 +224,12 @@ func newLibpodConfig(c *Config) error {
 		}
 	}
 
+	// hard code EventsLogger to "file" to match older podman versions.
```

Comment:
Sorry for what may be a dumb question: should this be (or is it) rootless only? If someone has an existing and modified `/usr/share/containers/libpod.conf`, would this clobber a journald setting?

Reply:
I think we should just stick with this.  I don't think this is a real issue with these customers.  And we want users to move off of libpod.conf anyways,